.docker/android_dev/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM ubuntu:22.04
-ARG NDK_VERSION=22.1.7171670
+ARG NDK_VERSION=23.1.7779620
```

Comment:
Did you try r25 too?

Reply:
@m-kuhn , I did not no, I thought it'd be better to match the NDK version with that used to compile Qt6 against.